AgContext.yaml

```diff
@@ -0,0 +1,149 @@
+
+agcontext_name:
+  type:
+    anyOf:
+    - type: string
+    description: |-
+      Agricultural context name
+      string. Required.
+      A string representing the name of the agricultural context.
+
+crop_type:
+  type:
+    anyOf:
+    - type: string
+    description: |-
+      General crop type
+      string. Required.
+      A string describing the general cropping scenario (e.g. "root vegetable").
```

Comment:
If this is an arbitrary string (not from a controlled vocabulary), is it useful?

Reply:
Currently it is arbitrary; I included this as a sort of place holder for a more controlled term that we could use - perhaps one from Guy/Michael?